index.html

```diff
@@ -32,10 +105,21 @@ <h2 class="card__heading">Question:</h2>
     </main>
     <footer>
       <nav class="nav">
-        <a class="nav__home" href="#"><img src="" alt="#" /></a>
-        <a class="nav__bookmarks" href="#"><img src="" alt="#" /></a>
-        <a class="nav__create" href="#"><img src="" alt="#" /></a>
-        <a class="nav__profile" href="#"><img src="" alt="#" /></a>
+        <a class="nav__menu" href="#"
+          ><img class="nav__menuimg" src="./images/house-solid.svg" alt="Home"
+        /></a>
+        <a class="nav__menu" href="#"
+          ><img class="nav__menuimg" src="./images/bookmark-solid.svg" alt="#"
+        /></a>
+        <a class="nav__menu" href="#"
+          ><img
+            class="nav__menuimg"
+            src="./images/clipboard-question-solid.svg"
+            alt="#"
+        /></a>
+        <a class="nav__menu" href="#"
+          ><img class="nav__menuimg" src="./images/user-solid.svg" alt="#"
+        /></a>
```

Comment:
If your icons are actually SVGs you can insert them directly with the `<svg>` tag instead of the `<img>`-tag